### PR TITLE
Pr/prefill data

### PR DESCRIPTION
**Pull request by eliatib**

### CODE REVIEW

File: src/Entity/ContactFormMessage.php [hunk 1]
```diff
@@ -22,45 +23,45 @@ class ContactFormMessage implements ResourceInterface
...
      */
     protected $message;
-
     /**
      * @ORM\Column(type="datetime")
      *
      * @var \DateTime|null
      */
     protected $sendTime;
+    /**
```
Comment (ondrej-kuhnel) [inline src/Entity/ContactFormMessage.php:66]: Add blank line

File: src/Controller/ContactFormController.php [hunk 2]
```diff
@@ -72,6 +79,18 @@ public function createContactMessage(Request $request): Response
     {
         $contact = new ContactFormMessage();
         $sitekey = $_ENV['RECAPTCHA_SITE_KEY'];
```
Comment (ondrej-kuhnel) [inline src/Controller/ContactFormController.php]: Resolve ENV variable in services.yml and hand over class constructor

**Review by ondrej-kuhnel — CHANGES_REQUESTED** [review]

File: src/Controller/ContactFormController.php (hunk 2 above)
Comment (eliatib) [reply src/Controller/ContactFormController.php]: https://github.com/mangoweb-sylius/SyliusContactFormPlugin/pull/4#discussion_r358761711

File: src/Entity/ContactFormMessage.php (hunk 1 above)
Comment (eliatib) [reply src/Entity/ContactFormMessage.php:66]: DONE

File: src/Controller/ContactFormController.php (hunk 2 above)
Comment (eliatib) [reply src/Controller/ContactFormController.php]: next issue (https://mangoweb.org/apps/sylius-test-plugin/issues/20)

Comment (eliatib) [reply src/Controller/ContactFormController.php]: https://github.com/mangoweb-sylius/SyliusContactFormPlugin/pull/6

Comment (ondrej-kuhnel) [reply src/Controller/ContactFormController.php]: what does it mean?

Comment (eliatib) [reply src/Controller/ContactFormController.php]: already resolve

**Review by ondrej-kuhnel — APPROVED** [review]

**Pull request merged**